src/libumf.c

```diff
@@ -97,3 +98,42 @@ void umfTearDown(void) {
 }
 
 int umfGetCurrentVersion(void) { return UMF_VERSION_CURRENT; }
+
+umf_result_t umfCtlGet(const char *name, void *ctx, void *arg, size_t size) {
+    if (name == NULL || arg == NULL) {
+        return UMF_RESULT_ERROR_INVALID_ARGUMENT;
+    }
+    // using ctx is disallowed for default settings
+    if (ctx && strstr(name, ".default.")) {
```

Comment:
please do this check in default subtree handler. This is ugly hack which will not work in some cases
i.e: umf.pool.byhandle.default.something